dart/lib/src/enricher/io_enricher_event_processor.dart

```diff
@@ -71,7 +69,6 @@ class IoEnricherEventProcessor extends EventProcessor {
   Map<String, dynamic> _getDartContext() {
     final args = Platform.executableArguments;
     final packageConfig = Platform.packageConfig;
-    final isolate = Isolate.current.debugName;
```

Comment:
This is only set on the `AndroidPlatformExceptionEventProcessor`.
So I'd not remove it, otherwise, the iOS metadata won't have the `Isolate.current.debugName` at all.

Reply:
No, this is also set when enabling the `attachThreads` option

https://github.com/getsentry/sentry-dart/blob/9835e1cc20cfb03785448ea1d2dd2e3297c2e544/dart/lib/src/sentry_client.dart#L185-L193

Reply:
Indeed, for some reason, VS Code only returned 1 result for `Isolate.current.debugName` search, all good then, thanks.